docs/officers/payments/accept.rst

```diff
@@ -1,46 +1,57 @@
-:og:description: While most members choose to pay online, cash and checks need to be recorded in Apiary manually by an officer or project manager. Treasurers also have the ability to apply waivers.
+:og:description: While most members choose to pay online, cash and checks must be recorded in Apiary manually by an officer or project manager. Officers also have the ability to apply waivers.
 
-Accepting Payments
+Accepting payments
 ==================
 
-While most members choose to pay online, cash and checks need to be recorded in Apiary manually by an officer or project manager.
-Treasurers also have the ability to apply waivers.
+.. vale Google.OxfordComma = NO
+.. vale Google.Passive = NO
+.. vale write-good.E-Prime = NO
+.. vale write-good.Passive = NO
+
+While most members choose to pay online, cash and checks must be recorded in Apiary manually by an officer or project manager.
+Officers also have the ability to apply waivers.
 
 .. hint::
    To record cash or check payments, you must have a :ref:`officer` or :ref:`project-manager` role.
    To apply waivers, you must have the :ref:`officer` role.
    If you need access, ask in :slack:`it-helpdesk`.
 
-Online-Only Payment Methods
----------------------------
+Online payment methods
+----------------------
+
+.. vale write-good.Weasel = NO
 
-The following payment methods are only accepted online and incur an additional surcharge depending on the amount of the transaction.
+The following payment methods are only accepted online and incur a surcharge depending on the amount of the transaction.
 
 - Credit and debit cards
 - Google Pay
 - Apple Pay
 - Cash App
 
-We do **not** accept payments via PayPal, Venmo, or Zelle (they're more complicated and expensive to process).
+Apiary **doesn't** support payments via PayPal, Venmo, or Zelle.
```

Comment:
I prefer the old phrasing. They're not accepted because RJ doesn't want to accept them. It's not just a technical limitation. The text in the parentheses is not necessary though.

Reply:
This was mostly an attempt to edit towards the [pronouns guidance](https://developers.google.com/style/pronouns#personal-pronouns), but I think a842b93bffba8bb0c3eb2560408329cc5d5ef73f is the clearest option.